app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/SubscriptionController.scala

```diff
@@ -65,7 +63,8 @@ class SubscriptionController @Inject()(
         _ <- EitherT(
               updateSession(sessionStore, request)(
                 _.copy(journeyStatus = Some(SubscriptionComplete(details, subscriptionResponse)))
```

Comment:
would this not be `JourneyStatus.Subscribed`? We don't really need `SubscriptionComplete` if we have that other journey status